color.d

```diff
@@ -1512,7 +1512,7 @@ struct Rectangle {
 
 	/// ditto
 	bool contains(in Point p) {
-		return (p.x >= left && p.y < right && p.y >= top && p.y < bottom);
```

Comment:
I don't like how you reorganized things, all that needed to change was the y to an x. I made the change on my local thing